CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## v5.13.0 (unreleased)
```

Comment:
if you want to release this fix as a patch build, then you have to mention the version as v5.12.1

Reply:
Yes, its a patch release. I will change it to v.5.12.1